database/config.py

```diff
@@ -0,0 +1,28 @@
+### config.py
+
+# Scheme: "postgres+psycopg2://<USERNAME>:<PASSWORD>@<IP_ADDRESS>:<PORT>/<DATABASE_NAME>" pragma: allowlist secret
+
+username = "username"  # pragma: allowlist secret
+
+password = "password"  # pragma: allowlist secret
+
+IPadd = "localhost"  # pragma: allowlist secret
+
+portNum = "1234"  # pragma: allowlist secret
+
+DBname = "ReleaseTask"  # pragma: allowlist secret
+
+DATABASE_URL = (
+    "postgresql+psycopg2://"  # pragma: allowlist secret
+    + username  # pragma: allowlist secret
+    + ":"
+    + password  # pragma: allowlist secret
+    + "@"
+    + IPadd  # pragma: allowlist secret
+    + ":"
+    + portNum  # pragma: allowlist secret
+    + "/"
+    + DBname  # pragma: allowlist secret
+)
+
+# print(DATABASE_URL)

```

Comment:
This was probably handy for debugging purposes, but now that we are about to merge our code into the master branch, let us remove any unnecessary print statements to keep our code base clean.